cms/models.py

```diff
@@ -472,6 +473,112 @@ def create_country_field(self, field, options):
         return ChoiceField(**options)
 
 
+class InstructorPage(Page):
+    """
+    Detail page for instructors.
```

Comment:
I think you can add in something like: https://github.com/mitodl/mitxonline/blob/e09ce6f99c0ed119e7a7aa31880a3cd8a02ff996/cms/models.py#L1525
Which will not allow child pages to be created for an InstructorPage.